src/main/java/soot/Soot.java

```diff
@@ -0,0 +1,42 @@
+package soot;
+
+import soot.manager.CommandManager;
+
+import java.util.Scanner;
+
+public class Soot {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+        CommandManager commandManager = new CommandManager();
+
+        boolean isBye = false;
+
+        drawLine(); //initial greeting
+        greetUser();
+
+        while (!isBye) {
+            line = in.nextLine(); //user input
+            drawLine();
+            isBye = commandManager.isInputBye(line);
+        }
+        greetGoodbye();
+    }
+
+    public static void drawLine() {
+        int LINE_LENGTH = 60;
```

Comment:
A variable should be named using camel form, only constants can be named as all caps.